packages/api/src/UserModule/UserModule.service.ts

```diff
@@ -114,11 +134,20 @@ export class UserModuleService {
     });
   }
 
-  async create(
-    userId: string,
-    moduleId: string
-  ) {
-    return this.userModuleRepository.create({ userId, moduleId }).save();
+  /**
+   * Find the last module completedAt the user completed
+   * @param userId User
+   */
+  async findLatestByUser(userId: string) {
```

Comment:
This function name is not clear that it's returning the `completedAt` value.
It could be renamed to `findLatestCompletedAtByUser`.

Reply:
Hey @shaneSheridan, this has been updated to findLatestCompletedAtByUser.